doc/cascadia/profiles.schema.json

```diff
@@ -869,15 +930,37 @@
         "icon": { "$ref": "#/definitions/Icon" },
         "name": {
           "description": "The name that will appear in the command palette. If one isn't provided, the terminal will attempt to automatically generate a name.",
+          "properties":{
+            "key":{
+              "type":"string"
+            }
+          },
           "type": [
             "string",
+            "object",
             "null"
           ]
+        },
+        "iterateOn":{
+          "type":"string",
+          "description":"used to create iterable commands based on other object on your settings. possible values:\n- \"profiles\" \n- \"schemes\"",
+          "enum":[
+            "profiles",
+            "schemes"
+          ]
+        },
+        "commands":{
+          "description": "list of commands to execute",
+          "items": {
+            "$ref": "#/definitions/Keybinding/properties/command"
+          },
+          "minItems": 1,
+          "type": "array"
         }
       },
-      "required": [
-        "command",
-        "keys"
+      "anyOf":[
+        {"required":["name","commands"]},
+        {"required":["command"]}
       ],
```

Comment:
I'm not sure this is right. Perhaps it should be like this:

- Require `name`, `keys`, or both, so that the action can be invoked.
- Also require `command` or `commands` (but not both), so that the action does something.

Reply:
Oh, I see this was already discussed and decided.